MVP Module/Source/Feed Presentation/FeedErrorViewModel.swift

```
@@ -0,0 +1,12 @@
+//
+//  FeedErrorViewModel.swift
+//  UIDesignPatternsChallenge
+//
+//  Created by Anton Ilinykh on 22.02.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+struct FeedErrorViewModel {
+	let message: String
```

The `message` could be an optional `String?`, so the Presenter can decide when to show/hide the message.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `errorView.display(FeedErrorViewModel(message: .none))` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's easier to manage state when all changes are centralized in a single component.